### DISCUSSION TITLE
clean names of target markdown chunks

### DISCUSSION THREAD

**Discussion by jmbuhr** [ropensci/targets#589] [Ideas]
Would it make sense to replace `-` in the names of target markdown chunks set up as `tar_simple` targets with `_` when they become targets to turn them into valid R variable names?

**Comment by wlandau** [comment]
Unfortunately, underscores in chunk labels still interfere with `bookdown`: https://github.com/ropensci/targets/pull/473#issuecomment-850383350, https://github.com/rstudio/bookdown/issues/1167. And I would prefer not all targets to use `tar_simple = TRUE` so that I can demonstrate the full range of functionality in Target Markdown.
